votor/src/event_handler.rs

```diff
@@ -206,13 +206,19 @@ impl EventHandler {
             // Received a parent ready notification for `slot`
             VotorEvent::ParentReady { slot, parent_block } => {
                 info!("{my_pubkey}: Parent ready {slot} {parent_block:?}");
+                assert_eq!(slot, first_of_consecutive_leader_slots(slot));
```

Comment:
if we perform a full cluster restart from a block in the middle of a leader window we'll need the initial ParentReady to kick things off which will not be the first of the window.